dataland-frontend/tests/component/testUtils/ExistenceChecks.ts

```diff
@@ -2,9 +2,11 @@
  * Checks if an image is present
```

Comment:
General question considering the concept of searching for the image/button within a parent-element:

Why do we suddenly need this?  Is it nor sufficient to just do "cy.get(...)" to get a specific button or image?

Reply:
I felt that that was neccessary in some intermediate state. I will remove it later.

Reply:
Ok. Leving this open as reminder

Reply:
Done